js/fc.js

```diff
@@ -1152,6 +1152,12 @@ var FC = {
             'RC Channel 14',        // 20
             'RC Channel 15',        // 21
             'RC Channel 16',        // 22
+            'Stabilized Roll+',     // 23
```

Comment:
Maybe we should change the lines with `Stabilised...` (note the `s` instead of `z` - UK spelling) a few lines up to `Stabilized` for consistency? I think most of the configurator code favors US spelling.